docs/source/rclone.rst

```diff
@@ -0,0 +1,420 @@
+    Choose a number from below, or type in your own value
+     1 / None
+       \ ""
+     2 / AES256
+       \ "AES256"
+    server_side_encryption> 2
+    The storage class to use when storing objects in S3.
+    Choose a number from below, or type in your own value
+     1 / Default
+       \ ""
+     2 / Standard storage class
+       \ "STANDARD"
+     3 / Reduced redundancy storage class
+       \ "REDUCED_REDUNDANCY"
+     4 / Standard Infrequent Access storage class
+       \ "STANDARD_IA"
+     5 / One Zone Infrequent Access storage class
+       \ "ONEZONE_IA"
+     6 / Glacier storage class
+       \ "GLACIER"
+     7 / Glacier Deep Archive storage class
+       \ "DEEP_ARCHIVE"
+     8 / Intelligent-Tiering storage class
+       \ "INTELLIGENT_TIERING"
+     9 / Glacier Instant Retrieval storage class
+       \ "GLACIER_IR"
+    storage_class> 1
+    Remote config
+    --------------------
+    [cgap_s3_submission]
+    type = s3
+    provider = AWS
+    env_auth = false
+    access_key_id =
+    secret_access_key =
+    region = us-east-1
+    endpoint =
+    location_constraint =
+    acl = private
+    server_side_encryption = AES256
+    storage_class =
+    --------------------
+    y) Yes this is OK
+    e) Edit this remote
+    d) Delete this remote
+    y/e/d> y
+
+If successful you should be able to transfer files from the local machine or your private
+s3 buckets into our submission bucket. If we have not told you the bucket name to submit
+to, please reach out to us at `cgap-support@hms-dbmi.atlassian.net <mailto:cgap-support@hms-dbmi.atlassian.net>`_.
+
+You then can transfer files from your local machine or your private S3 buckets into
+our submission bucket. Test our remote by filling out the accessioning spreadsheet
+and submitting it to our S3 bucket. Please denote a sensible case ID to use as a folder
+prefix for the current submission. Transfer the spreadsheet with:
+
+.. code-block:: bash
+
+    $ rclone copy ~/Documents/submit_cgap/case0001/case0001_accessioning.xls cgap_s3_submission:cgap-submission-bucket/case0001/case0001_accessioning.xls
+
+If successful, continue by transferring the raw files into the submission bucket.
+
+.. code-block:: bash
+    $ rclone copy cgap_s3_submission:your-private-s3-bucket/fastq1.fastq.gz cgap_s3_submission:cgap-submission-bucket/case0001/fastq1.fastq.gz
+    $ rclone copy cgap_s3_submission:your-private-s3-bucket/fastq2.fastq.gz cgap_s3_submission:cgap-submission-bucket/case0001/fastq2.fastq.gz
+
+If you are not using AWS S3 as your storage provider, see the following instructions.
+
+
+Transferring from Another Cloud
+-------------------------------
+
+As mentioned previously, `rclone` is a cross-platform compatible file transfer tool.
+This allows you to submit files to our S3 buckets that live in other cloud platforms.
+To do this, you will need to locate your cloud provider on the main `rclone documentation page <https://rclone.org/>`_
```

Comment:
I often prefer to put hyperlinks on their own line in part because they are hard to notice and in part because they're usually very verbose anyway and need a whole line.

```suggestion
To do this, you will need to locate your cloud provider on the main
`rclone documentation page <https://rclone.org/>`_
```